site/src/github/comparison_summary.rs

```diff
@@ -133,10 +133,40 @@ async fn summarize_run(
     } else {
         String::new()
     };
+
     let (inst_primary, inst_secondary) = inst_comparison
         .clone()
         .summarize_by_category(&benchmark_map);
 
+    let direction = inst_primary.direction().join(inst_secondary.direction());
+    let deserves_attention = deserves_attention_icount(&inst_primary, &inst_secondary);
+    let is_regression =
+        deserves_attention && matches!(direction, Direction::Regression | Direction::Mixed);
+
+    writeln!(
+        &mut message,
+        "### Overall result: {}{}\n",
+        direction.overall_result(),
+        if is_regression {
+            " - ACTION NEEDED"
+        } else {
+            ""
+        },
+    )
+    .unwrap();
+
+    let next_steps = if is_master_commit {
+        master_run_body(is_regression)
+    } else {
+        try_run_body(is_regression)
+    };
+    writeln!(&mut message, "{next_steps}\n").unwrap();
+
+    const DISAGREEMENT: &str =
+        "If you disagree with this performance assessment, please file an issue in \
+        [rust-lang/rustc-perf](https://github.com/rust-lang/rustc-perf/issues/new).";
```

Comment:
This was originally written when we moved to a new system for classifying regressions. As we've become more confident that the regression classification is not totally off base, I wonder if we should rephrase this to something more like "If you have questions about this performance assessment, please file an issue..."

Reply:
Or I could just remove it. Has anyone ever filed such an issue?

Reply:
Yes, though it's hard to tell if they filed an issue specifically because of this prompt or because they knew were to go already.